### ISSUE TITLE
Create FMC-Community letter of support

### ISSUE THREAD

**Issue by jonmatthis** [freemocap/2024-nsf-pose#19]
- [ ] Gather mini-letters of support from the FMC community (like, few sentences to a paragraph, with more space allotted to folks like @ajc27-git )

- [ ] Compile statements into a single 'Letter of Support' format (i.e. 2 pages max)
- [ ] Add Header saying "This is non-standard, but a better barometer of our current situation. We asked the POSE staff if it was ok, and they did not say No" 

- [ ] Incorporate into the Community survey that @trentwirth  and @aaroncherian ? Or do a separate solicitation? 
- [ ] Also should directly contact some key folks/students/etc

**Comment by jonmatthis** [comment]
The section about letters of collaboration from the FOA

```
1. Letters of Collaboration (required)
A minimum of three and up to five letters of collaboration from third-party users or contributors of the open-source product ust be uploaded as Supplementary Documents. These letters of collaboration must be from current users or contributors (who are not directly related to the proposing team) of the open-source product that is the subject of the proposed OSE. Each letter writer should clearly describe how they have contributed and will continue to contribute to the development of an OSE. These letters do not have to conform to the standard format specified in the PAPPG. In addition to the above  information, each letter of collaboration (not to exceed two pages) must include the name of the letter writer, current affiliations (institution or place of employment), and relationship to the members of the proposing team
```

But we have more flexibility than that. 

Should get at most a few sentences from each person (extra from folks like ajc) 

Info about - What they have used freemocap to do, what they plan to do, and generally comments to the effect that the freemocap OSE is an important and valuable thing.

**Comment by jonmatthis** [comment]
@aaroncherian will come up with a post and put it here. 

- [ ] Draft post for Discord 
- [ ] Draft instructions on Google Form

**Comment by aaroncherian** [comment]
Draft of the Discord post. Contains a link to the draft of the survey
---


:rotating_light: **Important: Call for Support!** :rotating_light:

Hey FreeMoCap Community!

We're excited to share that we're applying for the NSF Pathways to Enable Open-Source Ecosystems (POSE) grant. Grants like these help us continue to enhance and grow FreeMoCap. The POSE grant in particular focuses on helping organizations develop and sustain emerging open-source software projects by fostering community-driven development. This grant would enable us to build a robust managing organization for FreeMoCap, strengthen our community of contributors, and ensure FreeMoCap's long-term sustainability and societal impact. We could use your help!

**What We Need:**
We’re looking to gather mini-letters of support from our community. These will be compiled into a single letter of support for our grant application. We believe that hearing directly from the people who use and benefit from FreeMoCap will be extremely impactful. 

**What to Include:**
A few brief sentences covering:
- What you have used FreeMoCap for/what you plan to do with FreeMoCap
- Why having FreeMoCap as an open-source tool has been valuable for your work and the broader community

As we are planning on incorporating excerpts from the replies we receive into one form, *please keep your replies to just a few sentences*. Short and sweet will make it easier for us to draft the complete letter.

**Where to Submit**
Please submit your responses to this Google Form by **[INSERT DEADLINE HERE]**
https://forms.gle/vbLKUo5yYV8YSc2E8

**Comment by aaroncherian** [comment]
Changes in this edit:
Instruction changes:
- "This grant would help us continue developing a robust managing organization for FreeMoCap" - language changed to 'continue developing' so as not to imply we haven't been doing so all along
- 'mini-letters' changed to 'mini-statements', and the proceeding phrase regarding compiling the letters has been simplified for clarity
- Deadline added as Monday, August 26th
- Made it more explicit that we are taking excerpts of peoples' statements
- Added a thank you at the end because that's just nice to do

Survey changes:
- Changed hyperlink location to funding announcement
- Added question about age
- Changed 'Affiliation' to Background and made it a required question. Added within the question that participants are welcome to enter their affiliation in addition to their background, if desired
- Updated survey language to match updated Discord announcement post language mentioned above

----
:rotating_light: **Important: Call for Support!** :rotating_light:

Hey FreeMoCap Community!

We're excited to share that we're applying for the NSF Pathways to Enable Open-Source Ecosystems (POSE) grant. Grants like these help us continue to enhance and grow FreeMoCap. The POSE grant in particular focuses on helping organizations develop and sustain emerging open-source software projects by fostering community-driven development. This grant would help us continue developing a robust managing organization for FreeMoCap, strengthening our community of contributors, and ensuring FreeMoCap's long-term sustainability and societal impact. We could use your help!

**What We Need:**
We’re looking to gather mini-statements of support from our community, which we will compile together for our grant application. We believe that hearing directly from the people who use and benefit from FreeMoCap will be extremely impactful. 

**What to Include:**
A few brief sentences covering:
- What you have used FreeMoCap for/what you plan to do with FreeMoCap
- Why having FreeMoCap as an open-source tool has been valuable for your work and the broader community

*Please keep your replies to just a few sentences.* We will be taking excerpts from all submitted statements to compile into a 2-page document, so short and sweet will make it easier for us to draft the complete letter. 

**Where to Submit**
Please submit your responses to this Google Form by **Monday, August 26th**
https://forms.gle/vbLKUo5yYV8YSc2E8

Thanks everyone!

**Comment by aaroncherian** [comment]
@jonmatthis One question in the survey is what do you want in terms of collecting names or identifiers? I put in the instructions that they could enter their name or Discord usernames, but could totally see the argument for either just asking for names, or asking for first names or initials if that's more comfortable, and not asking for usernames. Just trying to find the line between what we would want on the letter and what info people would be comfortable giving out.

**Comment by jonmatthis** [comment]
we should mark the name field 'optional' and also let them tell us their username(s). 

tricky because we will need SOME identifier from everyone, but we don't really fully care about what kind.

Also, we're going to share this outside of Discord, so can't just say "name or Discord username" but we ALSO can';t just say "name or username" bc then folks will just drop a username and we won't know where they came from, unless we make a new intake form for each platform, which becomes a lot of work. 

We also can't just have "background" be a text box, because then folks will write a bunch of random stuff in there that will be hard to categorize - need to give a checkbox kind of thing (check this form for some examples: https://docs.google.com/forms/d/1phAduL-EoTANq8S9odMw4qwQYqAptXOZeM4UMcGjhaU/edit ) 

...but of course all of this needs to live in the context where the more info we ask about them, the more likely they are to bounce away. I think as long as we make sure that the demographics stuff takes significantly less than a minute to fill out, it should be OK

**Comment by aaroncherian** [comment]
Yeah I gotcha. I think ultimately, the deciding factor is how you envision the format of the statement. In my head, it looked something like this:


_[insert quote here]_
_from Joe Schmoe, amateur game dev, Bolivia_

_[insert next quote here]_
_from AM, movement researcher, Arctic Circle_



Which is what informed a lot of how I wrote the demographic questions. I wrote these questions envisioning what might make this specific line (*from Joe Schmoe, amateur game dev, Bolivia*) pop out more for each user. *If this is the format you also had in mind*, I think asking for a mandatory identifier (a first name or initials, whatever the user is comfortable with) would be good. And along this line of thinking, for this purpose I think the "Background" would be better as a textbox? It would make the responses/descriptions of our users more diverse and personalized, which could emphasize the variety of users that are working with FreeMoCap. 

However, if the above format is not what you had envisioned, or the goal is to gather demographic data for grant categorization, then a pre-defined list of background options would be more effective, yeah. Also worth that we still have the survey in mind to collect demographic data proper down the road.
